manager/controllers/packages.go

```diff
@@ -44,7 +45,6 @@ var queryItemSelect = database.MustGetSelect(&queryItem{})
 func packagesQuery(c *gin.Context, acc int) (*gorm.DB, error) {
 	subQ := database.SystemPackagesShort(database.Db, acc).
 		Select(queryItemSelect).
-		Where("sp.stale = false").
 		Group("spkg.name_id")
 
 	// We need to apply tag filtering on subquery
```

Comment:
It seems you've removed system_platform table joining, but unfortunately we need it to support tagging.

Reply:
I changed `isp` back to `sp` and it should work.